backend/Dockerfile

```diff
@@ -67,6 +70,9 @@ COPY ./backend/ ./
 # Copy everything from database
 COPY ./database/ ../database/
 
+# Copy prebuild @iota/client
+RUN cp -r /prebuild_node_modules ${WORKDIR}/node_modules
```

Comment:
This looks very unsafe - i'm willing to discuss this, but rather do it as before then like this.

Reply:
Ich habe ein paar Stunden verschiedene Wege probiert. Das war der einzige der funktioniert hat.
Du hast die Wahl.. diese Lösung oder längere Buildzeit.
Oder du präsentierst mir eine andere funktionierende Lösung.

Reply:
Dann empfehler ich längere Buildzeit - aber da können wir gerne im team drüber reden, was die Preferrenz ist.

Reply:
Okay. So groß ist der Unterschied ja da auch nicht. 
~10 min vorher, ~15 min nachher

Reply:
Bitte dies entfernen, längere Buildzeit macht was wir brauchen durch ``yarn install`` oder ``yarn`` haben wir dann in backend/node_modules/@iota/client... installiert